code/modules/mining/lavaland/necropolis_chests.dm

```diff
@@ -411,11 +411,11 @@
 	icon_state = "hook"
 	icon = 'icons/obj/lavaland/artefacts.dmi'
 	pass_flags = PASSTABLE
-	damage = 25
-	armour_penetration = 100
+	damage = 20
+	stamina = 20
```

Comment:
come on, at least make it do enough stamina damage to slow someone down

like, 45?

Reply:
you might also/alternatively want to make it knock people down?

Reply:
The entire point of the hook is that you get to drag people in to hit them with the other exceptionally effective weapons miners who go for gamer gear get, such as
Cleaving saw: 20 brute
Lava staff: 25 burn
Heirophant club: 15 brute, 15 burn
A basic pickaxe still does 15 damage, more than enough to slow.